gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/apim/infra/domain_service/api/ApiStateDomainServiceLegacyWrapper.java

```diff
@@ -73,11 +74,31 @@ public Api start(Api api, AuditInfo auditInfo) {
         return ApiAdapter.INSTANCE.fromApiEntity(started);
     }
 
+    @Override
+    public boolean startV2DynamicProperties(String apiId, EventManager eventManager) {
+        return apiStateService.startV2DynamicProperties(apiId, eventManager);
+    }
```

Comment:
We should inject `EventManager` in that class instead of providing it as a parameter.